lib/loader.js

```diff
@@ -30,8 +39,8 @@ module.exports = function(content, map) {
 
 	processCss(content, map, {
 		mode: moduleMode ? "local" : "global",
-		from: loaderUtils.getRemainingRequest(this),
-		to: loaderUtils.getCurrentRequest(this),
+		from: loaderUtils.getRemainingRequest(this).split("!").pop(),
```

Comment:
@bebraw why? what is bad use `loaderUtils.getRemainingRequest(this).split("!").pop()`, also https://github.com/webpack-contrib/css-loader/pull/532#discussion_r116260918